website/pages/en/index.js

```diff
@@ -78,29 +78,24 @@ const GetStarted = ({ language }) => {
         <a href={siteConfig.getPageUrl("videos.html", language)}>videos</a> on
         the people and concepts behind it.
       </p>
-      <p>
-        We&apos;re a small group of{" "}
-        <a href={siteConfig.getPageUrl("team.html", language)}>volunteers</a>{" "}
-        that spend their free time maintaining this project, funded by the
-        community. If Babel has benefited you in your work, becoming a{" "}
-        <a href="https://github.com/babel/babel/blob/master/CONTRIBUTING.md">
-          contributor
-        </a>{" "}
-        or <a href="https://opencollective.com/babel">sponsoring</a> might just
-        be a great way to give back!
-      </p>
     </div>
   );
 };
 
 const SponsorTier = props => {
-  const tierSponsors = siteConfig.sponsors.filter(
-    sponsor => sponsor.type == props.type && sponsor.tier === props.tier
-  );
+  let { min, max } = props;
+  const tierSponsors = siteConfig.sponsors.filter(sponsor => {
+    let value = Math.max(sponsor.monthly, sponsor.yearly / 12);
```

Comment:
Figured this could account for if you are currently donating per month, or if you just started this month, or stopped but still donated an equivalent amount the past year?